packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -801,7 +801,7 @@ export default class TabAdvanced extends Component {
                 <tr>
                   <th>
                     {_('autoPowerOn')}
-                    {vm.auto_poweron && !vmPool.auto_poweron && (
+                    {vm.auto_poweron && !vmPool?.auto_poweron && (
```

Comment:
IMO, we should display the warning, only if `vmPool?.auto_poweron === false`

Reply:
Or we could write it as `vmPool !== undefined && !vmPool.auto_poweron`